src/bot/commands/joke.ts

```diff
@@ -41,7 +42,7 @@ export default class JokeCommand extends Command {
         {
           color: Colors.PRIMARY,
           title: blague.joke,
-          description: `|| ${blague.answer} ||`,
+          description: spoiler(blague.answer),
```

Comment:
Compte tenu du fait qu'on n'aura pas les espaces et le fait qu'ajouter un templating autour de answer pour le repasser dans la fonction ne fera pas propre, je propose de rollback.
```suggestion
          description: `|| ${blague.answer} ||`,
```

Reply:
@DraftProducts Malgré les espaces mis en place au niveau du code, cela ne se voit pas du tout coté discord donc je pense que l'on peut laisser la fonction spoiler.

Reply:
On voit la différence sur Desktop 🤔

Reply:
Je ne vois pas du tout la différence sur Desktop 😕